src/transaction.ts

```diff
@@ -11,13 +11,17 @@ export class Transaction implements IModel  {
   public amount: number;
   public date: Date;
   public signature: string;
+  public contractId: Number;
```

Comment:
Please change this to `number` with a lowercase.
Please also change `contractId` to `smartContractId`.